src/main/java/net/ccbluex/liquidbounce/features/module/modules/movement/flymodes/other/Collide.kt

```diff
@@ -0,0 +1,22 @@
+package net.ccbluex.liquidbounce.features.module.modules.movement.flymodes.other
+
+import net.ccbluex.liquidbounce.event.BlockBBEvent
+import net.ccbluex.liquidbounce.features.module.modules.movement.flymodes.FlyMode
+import net.minecraft.util.AxisAlignedBB
+
+object Collide : FlyMode("Collide") {
+    override fun onBB(event: BlockBBEvent) {
+        event.boundingBox = AxisAlignedBB(
```

Comment:
This is weird...

Reply:
It's the best I could do, like converting old code to new code, yeah...

Reply:
Elaborate

Reply:
You usually offset by -2, 2, ... and so on, not mentioning that the offset makes it unclear at first sight.